clinica/pipelines/cli_param/option.py

```diff
@@ -185,3 +185,10 @@
     type=click.Path(exists=True, dir_okay=True, resolve_path=True),
     help="Compute atlases at the end of the path",
 )
+
+light_version = option(
+    "-li",
+    "--light_version",
+    is_flag=True,
+    help="Use a lighter version of the pipeline",
```

Comment:
We can discuss this, but this feels unclear for users.
Maybe something like `--delete_cache`, or `--clean_cache`?

Reply:
Yes, I agree. We should discuss it ! (Not only for the naming, for the logic as well) :)